Installer/scripts/products.iss

```diff
@@ -12,48 +13,56 @@ de.depdownload_msg=Die folgenden Programme werden benötigt bevor das Setup fort
 fr.depdownload_msg=Pour fonctionner cette application a besoin de :%n%n%1%nTélécharger et installer maintenant?
 es.depdownload_msg=Las siguientes aplicaciones son requeridas para que el instalador pueda continurar:%n%n%1%n¿Descargar e instalarlo ahora?
 it.depdownload_msg=Prima che l'installazione possa proseguire sono richieste le seguente applicazioni :%n%n%1%nVuoi scaricarle e installarle?
+pt_br.depdownload_msg=As seguintes aplicações são necessárias antes que a instalação possa continuar:%n%n%1%nBaixar e instalar agora?
```

Comment:
The fr and it strings have an additional space before the ":", please remove the leading space from the other strings so they are equal.